openverse_catalog/dags/data_refresh/data_refresh_task_factory.py

```diff
@@ -138,37 +140,77 @@ def create_data_refresh_task_group(
             pool=DATA_REFRESH_POOL,
         )
 
-        data_refresh_post_data = {
-            "model": data_refresh.media_type,
-            "action": "INGEST_UPSTREAM",
+        # This UUID is the suffix for the new index created as a result of this refresh.
+        index_suffix = str(uuid.uuid4())
+
+        def _get_http_operator(task_id: str, post_data: dict) -> SimpleHttpOperator:
+            """
+            Get a ``SimpleHttpOperator`` instance that is configured to make a POST
+            request with the given POST data.
+
+            :param task_id: the name of the task associated with the operator
+            :param post_data: the JSON data to send to the ingestion server
+            """
+
+            return SimpleHttpOperator(
+                task_id=task_id,
+                http_conn_id="data_refresh",
+                endpoint="task",
+                method="POST",
+                headers={"Content-Type": "application/json"},
+                data=json.dumps(post_data),
+                response_check=lambda response: response.status_code == 202,
+                response_filter=response_filter_data_refresh,
+            )
+
+        def _get_http_sensor(task_id: str, endpoint: str) -> HttpSensor:
+            """
+            Get an ``HttpSensor`` instance that waits for a given  ingestion server task
+            to be completed. The trigger task status can be observed by polling an
+            endpoint.
+
+            :param task_id: the name of the task associated with the sensor
+            :endpoint: the REST endpoint for tracking the status of the triggered task
+            """
+
+            return HttpSensor(
+                task_id=task_id,
+                http_conn_id="data_refresh",
+                endpoint=endpoint,
+                method="GET",
+                response_check=response_check_wait_for_completion,
```

Comment:
`response_check_wait_for_completion` is failing because it expects a `percent_completed` in the response, which it looks like should now be `progress`.

Obviously not a problem with this PR, but it might also be an opportunity to update that [error log](https://github.com/WordPress/openverse-catalog/blob/main/openverse_catalog/dags/data_refresh/data_refresh_task_factory.py#L91): it doesn't actually signify an error _triggering_ the data refresh, but during the task execution on the ingestion server. I've been meaning to update that one for awhile! 😄

Reply:
Updated the log messages in f099edf.